service/src/main/java/org/ehrbase/service/StatusServiceImp.java

```diff
@@ -33,7 +33,7 @@
 public class StatusServiceImp implements StatusService {
 
     @SuppressWarnings("SpringJavaAutowiredFieldsWarningInspection")
-    @Autowired
+    @Autowired(required = false)
```

Comment:
The annotation does not have to be removed

Reply:
ups ... slipped in, was from a local test